charts/aws-fsx-csi-driver/templates/node-daemonset.yaml

```diff
@@ -1,32 +1,38 @@
 apiVersion: apps/v1
```

Comment:
Since this is a daemonset and meant to run on every node it should probably have some default tolerations with the ability to set more.  EBS handles it like this:
```
      tolerations:
        {{- if .Values.node.tolerateAllTaints }}
        - operator: Exists
        {{- else }}
        - key: CriticalAddonsOnly
          operator: Exists
        - operator: Exists
          effect: NoExecute
          tolerationSeconds: 300
        {{- end }}
        {{- with .Values.node.tolerations }}
        {{- toYaml . | nindent 8 }}
        {{- end }}
```